_posts/2012-04-18-app.markdown

```diff
@@ -85,15 +85,24 @@ cd railsgirls
 {% endhighlight %}
 
     <div>
-<p>Wenn Du nun in diesem Verzeichnis <code>ls</code> aufrufst, siehst Du Verzeichnisse wie <code>app</code> und <code>config</code>. Du kannst den Railsserver wie folgt starten:</p>
-    </div>
+<p>Wenn Du nun in diesem Verzeichnis <code>ls</code> aufrufst, siehst Du Verzeichnisse wie <code>app</code> und <code>config</code>.</p>
+
+<p>Wenn du Codenvy benutzt, oder den Fehler "Gem Load Error is: Could not find a JavaScript runtime" erhältst, musst du folgendes in die zweite Zeile der Datei `Gemfile` eintragen und anschließend den Befehl "bundle" im Terminal ausführen:</p>
```

Comment:
Das sollten wir auf jeden Fall in der Installationsanleitung unterbringen, weil vielleicht nicht alle nach der Installation/Einrichtung genau dieses tutorial nehmen sondern vielleicht ein anderes beginnen. Ob wir es dann hier nochmal erwähnen, da bin ich leidenschaftslos.